optimum/intel/openvino/quantization.py

```diff
@@ -387,7 +387,13 @@ def _quantize_ovbasemodel(
                     # The model may be for example OVModelForImageClassification, OVModelForAudioClassification, etc.
                     self.model.model = _hybrid_quantization(self.model.model, quantization_config, calibration_dataset)
             else:
-                _weight_only_quantization(self.model.model, quantization_config, calibration_dataset)
+                if is_diffusers_available() and isinstance(self.model, OVStableDiffusionPipelineBase):
+                    sub_model_names = ["vae_encoder", "vae_decoder", "text_encoder", "text_encoder_2", "unet"]
+                    sub_models = filter(lambda x: x is not None, (getattr(self.model, name) for name in sub_model_names))
```

Comment:
```suggestion
                    sub_models = filter(lambda x: x, (getattr(self.model, name) for name in sub_model_names))
```

Will help with keeping this a single line after `make style`